templates/test/dev/custom-builds-machine-pool/kustomization.yaml

```diff
@@ -1,32 +1,35 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 namespace: default
 resources:
-  - ../../../test/ci/prow-machine-pool
-patchesStrategicMerge:
-  - ../patches/control-plane-custom-builds.yaml
-  - patches/machine-pool-deployment-pr-version-windows.yaml
-  - patches/custom-builds.yaml
+- ../../../test/ci/prow-machine-pool
 patches:
-  - target:
-      group: controlplane.cluster.x-k8s.io
-      version: v1beta1
-      kind: KubeadmControlPlane
-      name: .*-control-plane
-    path: ../../../test/ci/prow-ci-version/patches/oot-credential-provider-kcp.yaml
-  - target:
-      group: bootstrap.cluster.x-k8s.io
-      version: v1beta1
-      kind: KubeadmConfig
-      name: .*-mp-win
-      namespace: default
-    path: patches/kubeadm-bootstrap-machine-pool-windows-k8s-pr-binaries.yaml
+- path: ../patches/control-plane-custom-builds.yaml
+- path: patches/machine-pool-deployment-pr-version-windows.yaml
+- path: patches/custom-builds.yaml
+- path: ../../../test/ci/prow-ci-version/patches/oot-credential-provider-kcp.yaml
```

Comment:
Since the `sortOrder` is set to `fifo`, a change in the order of listing these patch files generates a different `cluster-template-custom-builds-machine-pool.yaml`